### ISSUE TITLE
Member pre-registration is off-putting

### ISSUE THREAD

**Issue by pwilkins** [mitodl/lore#864]
Currently users must "pre-register" on the site for authentication to work.  This step gives users the impression that the system is broken.  Better to integrate invitation, registration, and log in.

> _Source: Mary Ellen Wiltrout's Notes_ 
> https://docs.google.com/document/d/10iJdumDyhArCXjZVMVAtcYmpCzzqZYcsI7B9LxDg0m8/edit?pref=2&pli=1

@pdpinch

**Comment by pdpinch** [comment]
Agreed the best way to do this is to have a "pending account" so that admins can invite users that don't yet have accounts and they will be granted permission when their account is created.
